Runtime/Common/GuidHelper.cs

```diff
@@ -23,5 +25,20 @@ public static bool IsNullOrEmpty(string guid)
             const string emptyGuid = "00000000-0000-0000-0000-000000000000";
             return string.IsNullOrEmpty(guid) || guid == emptyGuid;
         }
+
+        /// <summary>
+        /// Converts a random string into a guid representation.
+        /// </summary>
+        public static Guid FromString(string value)
```

Comment:
Can we rename this to `FromMD5String`? `FromString` suggests that a GUID string can be provided here.

Reply:
MD5 is an algorithm behind the scene that we're using to convert string into bytes that later we can use to generate GUID. It doesn't make sense to include MD5 in the name in my opinion